lib/storage/providers/NativeFileHandler.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * The native FS handler
+ * We store files in our app's directory space, which does not require special
+ * permission
+ */
+
+import * as FileSystem from 'expo-file-system';
```

Comment:
I saw `react-native-unimodules` are removed from Expensify/App. Unfortunately Expo-File-System won't work without them (it actually needs the new package `expo-modules`)...

I see the following options 
1. Removing `prepareFile` means we'll not be moving files to "our space" so we also won't need to delete files from our space and it seems we won't need any file handling
2. Replace `expo-file-system` with `react-native-fs`, I've opted for `expo-file-system` because we had already installed `react-native-unimodules` at the time and it was included there. Also I think as an Expo package it would have better support
3. Keep using `expo-file-system` which means installing `expo-modules` in Expensify/App